docs/content/resources/docker.container.md

```diff
@@ -1,197 +0,0 @@
----
-title: "docker.container"
```

Comment:
All of the docs are removed. Running `make` in /docs fails with `unknown command "gen" for "converge"`, but using the new `gen docs` command places docs in /markdown-docs.